engines/python/setup/djl_python/rolling_batch/lmi_dist_rolling_batch.py

```diff
@@ -63,15 +61,27 @@ def __init__(self, model_id_or_path, device, properties, **kwargs):
         self.cache: Batch = None
 
     def _init_model(self, kwargs, model_id_or_path):
-        self.config = AutoConfig.from_pretrained(model_id_or_path,
-                                                 **kwargs)
-        self.batch_cls = get_batch_cls_from_architecture(self.config.architectures[0])
+        self.config = AutoConfig.from_pretrained(model_id_or_path, **kwargs)
+        self.batch_cls = get_batch_cls_from_architecture(
+            self.config.architectures[0])
         sharded = int(self.properties.get("tensor_parallel_degree", "-1")) > 1
-        self.model = get_model(model_id_or_path,
-                               revision=None,
-                               sharded=sharded,
-                               quantize=None,
-                               trust_remote_code=kwargs.get("trust_remote_code"))
+        quantize = self.properties.get("quantize", None)
```

Comment:
in the original properties, we have option.load_in_8bit, can we reuse this param?

Reply:
`load_in_8bit` is a boolean. Assuming we add `gptq` support in next release we need a parameter which can take the quantization algo name instead of just a boolean. @lanking520 thoughts?